fix `npm install` issue for the client

```
client % npm install
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: client@1.0.0
npm error Found: react@18.3.1
npm error node_modules/react
npm error   react@"^18.3.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@"^16.8.0 || ^17.0.0" from @material-ui/core@4.12.4
npm error node_modules/@material-ui/core
npm error   @material-ui/core@"^4.12.3" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /Users/hideya/.npm/_logs/2024-12-04T07_14_05_023Z-eresolve-report.txt
npm error A complete log of this run can be found in: /Users/hideya/.npm/_logs/2024-12-04T07_14_05_023Z-debug-0.log

```

--- a/client/src/App.tsx
+++ b/client/src/App.tsx
@@ -1,7 +1,7 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { ThemeProvider, createTheme } from '@material-ui/core/styles';
-import CssBaseline from '@material-ui/core/CssBaseline';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
 import ChatScreen from './components/ChatScreen';
 import Login from './components/Login';
 import Register from './components/Register';

